botocore/client.py

```diff
@@ -183,10 +183,13 @@ def _get_client_args(self, service_model, region_name, is_secure,
 
         endpoint_creator = EndpointCreator(self._endpoint_resolver,
                                            region_name, event_emitter)
+        timeout = (client_config.connect_timeout, client_config.read_timeout) \
```

Comment:
If you're going to break this into multiple lines, let's just write this as a standard `if` statement.  The main appeal of these conditional expressions is putting them on a  single line.